### DISCUSSION TITLE
GraphqL Playground to be available even when not connected to the net

### DISCUSSION THREAD

**Discussion by VinayaSathyanarayana** [keystonejs/keystone#5705] [Feature requests]
Currently GraphQL Playground is  not available when not connected to the Internet. This hinders development when one is offline

GraphQL Playground should be available even when one is offline

**Comment by bladey** [comment]
Hi @VinayaSathyanarayana, this is specific to the GraphQL Playground package, which has been requested here - https://github.com/graphql/graphql-playground/issues/1281
